### DISCUSSION TITLE
No export for slots on "import { Slot } from '@blocksuite/store';"

### DISCUSSION THREAD

**Discussion by TQuantab** [toeverything/blocksuite#5885] [Q&A]
Hi =)
im currently trying to create my own slots, but the descriped export from the DOCS is not available :/

Is this a Bug?

**Comment by doodlewind** [comment]
Should be `import { Slot } from '@blocksuite/global/utils'`, PR welcome for documentation corrections!

Edit: also exported in `@blocksuite/store` now.

**Comment by TQuantab** [comment]
Hi, thanks for the correction =)

Is it possible to add additional (custom) slots to the model, as in:

```js
model.propsUpdated.on(() => updateMyComponent());
```
